src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -407,7 +412,7 @@ function WorkspaceMembersPage(props) {
         >
             {({safeAreaPaddingBottomStyle}) => (
                 <FullPageNotFoundView
-                    shouldShow={(_.isEmpty(props.policy) || !Policy.isPolicyOwner(props.policy)) && !props.isLoadingReportData}
+                    shouldShow={(_.isEmpty(props.policy) || !Policy.isPolicyOwner(props.policy)) && !props.isLoadingReportData && !props.isLoadingPolicyMembers}
```

Comment:
But as far as I understand `props.isLoadingReportData` should also handle policies loading, because when opening the app for the first time, the `policyParams` is an empty object and adding another onyx key would be redundant.

Reply:
@aimane-chnaif

Reply:
I can't reproduce this:
https://github.com/Expensify/App/assets/91068263/5dd33768-cf3d-4ee1-b9f5-586dbdafbd2d